### PR TITLE
About design

### PR DESCRIPTION
**Pull request by v-nakon**

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -222,3 +222,70 @@
...
+  height: 81px;
+  font-family: 'Fira Sans';
+  line-height: 28px;
+  font-size: 18px;
+  color: #222222;
+  opacity: 0.7;
+  margin-top: 30px;
+}
+
+.margin-for-about {
```
Comment (byzyk) [inline src/style.css]: why do we need this class?

File: src/style.css [hunk 2]
```diff
@@ -222,3 +222,70 @@
...
+  display: flex;
+  justify-items: auto;
+  width: 285px;
+  height: 65px;
+  margin-top: 61px;
+}
+
+.about-name {
+  width: 189px;
+  height: 26px;
```
Comment (byzyk) [inline src/style.css]: same as above

File: src/style.css [hunk 3]
```diff
@@ -222,3 +222,70 @@
...
+  font-weight: 500;
+  line-height: 32px;
+  font-size: 22px;
+  color: #222222;
+  margin-left: 30px;
+}
+
+.about-position {
+  width: 189px;
+  height: 21px;
```
Comment (byzyk) [inline src/style.css]: same as above

File: src/style.css (hunk 1 above)
Comment (v-nakon) [reply src/style.css]: because we have two `div` with same properties, but for second `div` need another `margin-top`.

**Pull request merged**